core/arch/arm/mm/core_mmu.c

```diff
@@ -444,6 +436,28 @@ static bool __maybe_unused is_map_tee_ram(struct tee_mmap_region *mm)
 	return (mm->type == MEM_AREA_TEE_RAM);
 }
 
+static bool is_map_flat_mapped(struct tee_mmap_region *mm)
```

Comment:
Function names aren't supposed to be questions, rather answers to questions. I'd prefer `map_is_flat_mapped()`.